src/sid/contacts.py

```diff
@@ -618,7 +620,8 @@ def post_process_contacts(contacts, states, contact_models):
 
     # Dead people and ICU patients don't have contacts.
     has_no_contacts = states["needs_icu"] | states["dead"]
-    contacts.loc[has_no_contacts] = 0
+    contacts.loc[has_no_contacts, random_models] = 0
+    contacts.loc[has_no_contacts, recurrent_models] = False
```

Comment:
Sorry, I thought the zero is automatically converted to False.

Reply:
Yes, that is right. We did this, trying to solve the typing Error we had that turned out to be caused elsewhere and left it in. It shouldn't make a difference.